lib/msf/core.rb

```diff
@@ -14,8 +14,15 @@
 require 'rex'
 require 'rex/ui'
 require 'rex/arch'
+
 include Rex::Arch
 
+NTLM_CONST   ||= ::Rex::Proto::NTLM::Constants
+NTLM_CRYPT   ||= ::Rex::Proto::NTLM::Crypt
+NTLM_UTILS   ||= ::Rex::Proto::NTLM::Utils
+NTLM_BASE    ||= ::Rex::Proto::NTLM::Base
+NTLM_MESSAGE ||= ::Rex::Proto::NTLM::Message
```

Comment:
In a future PR we'll nuke these aliases, as well as the `include Rex::Arch` approach too

Reply:
This approach is an alternative, defining the aliases within a base class:

https://github.com/rapid7/metasploit-framework/blob/710ed31b75618af8ce0bdb19480a0a88871b0e69/lib/rex/proto/smb/client.rb#L21-L29